ios/ReactTestApp/ReactInstance.swift

```diff
@@ -97,6 +97,13 @@ final class ReactInstance: NSObject, RCTBridgeDelegate {
     }
 
     func initReact(onDidInitialize: @escaping () -> Void) {
+        if bridge == nil {
+            NotificationCenter.default.post(
+                name: .ReactTestAppPreInitReactNativeInstance,
+                object: []
```

Comment:
I think you can pass `nil` here:
```suggestion
                object: nil
```